### PR TITLE
Fixed FindBugs issues

### PR DESCRIPTION
**Pull request by rsandell**
And configured FindBugs for xml output so Jenkins can pick it up.

@reviewbybees

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
:bee:

File: src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java [hunk 1]
```diff
@@ -66,8 +66,9 @@ public String getScript() {
         FilePath shf = c.getScriptFile(ws);
 
         String s = script;
-        if (!s.startsWith("#!")) {
-            String defaultShell = Jenkins.getInstance().getInjector().getInstance(Shell.DescriptorImpl.class).getShellOrDefault(ws.getChannel());
+        final Jenkins jenkins = Jenkins.getInstance();
+        if (!s.startsWith("#!") && jenkins != null) {
```
Comment (KostyaSha) [inline src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java]: Just copy static `getJenkinsInstance()` implementation into some Util class or in this class.

**Comment by stephenc** [comment]
@reviewbybees done

**Comment by oleg-nenashev** [comment]
:bee: 
Merging the PR

**Pull request merged**